students/briannagata/session02/fizzbuzz.py

```diff
@@ -0,0 +1,18 @@
+# FizzBuzz exercise
+# Python 3.6
+
+
+def fizzbuzz():
+    """
+    From an inclusive range from 1 - 100, print whether the number is a
+    multiple of 3, 5, or both and print the associated Fizz/Buzz words.
+    """
+    for cycle in range(1, 101):
+        if cycle % 3 is 0 and cycle % 5 is 0:
```

Comment:
You do not wnat to use "is" for this test.

"is" tests whether the objects are actually the same object --not whether they have the same value. AS you can easily have multiple objects that happen to have the same value, "is" will fail in the general case.

This works because cpython has an optimization called "interning" -- since small integers are used so often, the interpreter keeps a pool if them around to re-use, rather than creating multiple ints with the same value.

so "is" will work as a test for small integers, but not large ones:

```
In [62]: x = 345678

In [63]: y = 345678

In [64]: x is y
Out[64]: False
```

You really want if cycle % 3 == 0 and cycle % 5 == 0: